src/Admin/AdherentAdmin.php

```diff
@@ -300,8 +300,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                         return;
                     }
 
-                    $qb->andWhere(sprintf('%s.postAddress.postalCode', $alias).' LIKE :postalCode');
-                    $qb->setParameter('postalCode', $value['value'].'%');
+                    $value = array_map('trim', explode(',', strtolower($value['value'])));
```

Comment:
what about move this code into a class (ex `PostalCodeQueryBuilderUtils`) with static method for doing the stuff for avoiding the duplication?
`'callback' => [PostalCodeQueryBuilderUtils::class, 'doStuff']`

Reply:
in this case a custom `AdminExtension` should be appropriate: 
https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/extensions.html